corehq/apps/analytics/signals.py

```diff
@@ -56,6 +60,7 @@ def update_subscription_properties_by_user(couch_user):
             properties[edition] = "yes"
```

Comment:
Wow, this gets called every time a couch user is saved? That seems like a lot of postgres hits... 
I know you didn't write it, just remarking.